### ISSUE TITLE
it is conceptually mistake to use log loss for performance comparison 

### ISSUE THREAD

**Issue by Sandy4321** [manujosephv/pytorch_tabular#252]
**Describe the bug**
it is conceptually mistake to use log loss for performance comparison 
 since 
1
lightgbm as all gmb models
 (  maybe except https://stanfordmlgroup.github.io/projects/ngboost/ NGBoost: Natural Gradient Boosting for Probabilistic Prediction
Tony Duan*, Anand Avati*, Daisy Yi Ding, Sanjay Basu, Andrew Ng, Alejandro Schuler  ) 
are not good in prediction probabilities
2
then same should be carefully tested for deep NN

3
may you change provement that your code is performing , for example 
https://github.com/manujosephv/pytorch_tabular/blob/main/examples/PyTorch%20Tabular%20with%20Bank%20Marketing%20Dataset.ipynb
to compare actual confusion matrices pls 
I did some attempt 
preds_GATE_Full  = tabular_model.predict(test)
but what is the way to calculate ground truth predictions from your models  
![image](https://github.com/manujosephv/pytorch_tabular/assets/11426119/1b7225a9-92ed-4c2f-a2ff-2074d9bb4c3a)

4
where is description of used algorithms , especially for categorical data values preprocessing ?

**Comment by Sandy4321** [comment]
ok , i found how to calculate performance 
![image](https://github.com/manujosephv/pytorch_tabular/assets/11426119/9456618f-0e9e-4ec0-82c3-5dbb210f6b90)

**Comment by Sandy4321** [comment]
so
confusion_matrix for GATE_Full
array([[7765,  220],
       [ 653,  405]], dtype=int64)
confusion_matrix for lightgbm
array([[7684,  301],
       [ 528,  530]], dtype=int64)

when modle training time much less for lightgbm ( I used cpu runs)
what is about inference time?
what is about ram and cpu/gpu resources use?
